### PR TITLE
reverse words of a string

### PR DESCRIPTION
**Pull request by Ganesh-Agarwal-25**
This PR is for Hacktober fest 2020.

# Welcome to Dart community

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Dart/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Dart files are placed inside an existing directory.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Ganesh-Agarwal-25** [comment]
@Parowicz please review this program for hacktober 2020.

File: strings/reverse_words_of_string.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
+String reverseStringWords(String s) {
```
Comment (Parowicz) [inline strings/reverse_words_of_string.dart:1]: You could add some tests
```suggestion
import 'package:test/test.dart';

String reverseStringWords(String s) {
```

File: strings/reverse_words_of_string.dart [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        res += s[j1];
+      }
+    }
+  }
+  return res;
+}
+
+void main() {
+  String s = "abhishek.is.a.good.boy";
+  print(reverseStringWords(s));
```
Comment (Parowicz) [inline strings/reverse_words_of_string.dart:23]: For example
```suggestion
  test("reverseStringWords single word returns same word", () {
    expect(reverseStringWords("word"), equals("word"));
  });

  test("reverseStringWords w1.w2 returns w2.w1", () {
    expect(reverseStringWords("w1.w2"), equals("w2.w1"));
  });

  test("reverseStringWords on empty string returns empty string", () {
    expect(reverseStringWords(""), equals(""));
  });

  test("reverseStringWords", () {
    expect(reverseStringWords("abhishek.is.a.good.boy"),
        equals("boy.good.a.is.abhishek"));
  });
```

**Comment by Parowicz** [comment]
LGTM!

**Pull request merged**